templates/taxi/manufacturer_form.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+{% block content %}
+  <h1> {{ object|yesno:"Update the,Create new" }} manufacturer </h1>
+  <form action="" method="post" novalidate>
+    {% csrf_token %}
+    {{ form|crispy }}
+    <input type="submit" value="Submit">
```

Comment:
It seems you have forgotten to add a `Cancel` button. The `Cancel` button should lead to the previous page the user was on. This is a requirement from the checklist. Please, add it.